lib/panoptes-subject-uploader.iced

```diff
@@ -29,6 +29,27 @@ findImages = (searchDir, metadata) ->
         imageFiles.push existingImageFile
   imageFiles
 
+findImagesURLs = (metadata) ->
+  imageURLs = []
+  for key, value of metadata
+    httpsRegexPattern = new RegExp("^(http|https)://", "i")
+    fileRegexPattern = new RegExp("([^\/]+\.(?:jpg|jpeg|gif|png|svg|mp4|txt))$", "i")
+    if httpsRegexPattern.test(value) && fileRegexPattern.test(value)
```

Comment:
This is nitpicky and for consistency with the syntax since this is in coffeescript: Maybe `and` instead of `&&`